src/groupchats.h

```diff
@@ -85,6 +85,9 @@ void groupchat_onGroupModeration(ToxWindow *self, Toxic *toxic, uint32_t groupnu
 
 void groupchat_rejoin(ToxWindow *self, Toxic *toxic);
 
+/* Updates the groupchat topic in the top statusbar. */
+void groupchat_update_statusbar_topic(ToxWindow *self, Tox *tox);
```

Comment:
`const Tox *`? Shouldn't need a mutable `Tox` to get the topic.

Reply:
True, that Tox pointer needs to be const'd all over toxic's codebase.